frontend/src/views/SettingWorkspaceLabels.vue

```diff
@@ -0,0 +1,139 @@
+<template>
```

Comment:
SettingWorkspaceLabel instead of SettingWorkspaceLabels

we are using singular form because plural is hard